scripts/mailbox-prep.sh

```diff
@@ -92,10 +103,4 @@ set -e
 
 kubectl ws "$mbws"
 
-scriptdir="$(dirname "$0")"
-
-"$scriptdir/ensure-syncer-plugin.sh"
-
-kcpdir="$scriptdir/../build/syncer-kcp"
-
-"$kcpdir/bin/kubectl-kcp" workload edge-sync "$stname" --syncer-image "$syncer_image" -o "$output"
+"$bindir/kubectl-kcpforedgesyncer" workload edge-sync "$stname" --syncer-image "$syncer_image" -o "$output"
```

Comment:
Because this script does not know what directory is current when it is invoked and, I think for the convenience of users (who generally have to manage the output files), should not mandate a particular directory be current.

Were you thinking that `$(cd ...)` has a lasting side-effect?  It does not, the stuff inside the parens is executing in a sub-shell.